docs/DeveloperGuide.adoc

```diff
@@ -296,6 +296,29 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 (For all use cases below, the *System* is `Cooking Papa` and the *Actor* is the `user`, unless specified otherwise)
 
+[source,shell]
+----
+Use Case: UC02 - View Recipes
+
+MSS:
+1. User chooses to view recipes.
+2. User enters the index of the desired recipe.
```

Comment:
2 makes 2a and 2b sound funky. Maybe it would be better to change 2 to: User enters input. Then we can add 2c, which is User enters index of the desired recipe.

Reply:
Also, view does not have parameters for tags and keyword.

Reply:
My plan was to make viewing of the recipes as the key feature, followed by the searching as the extension. Index will straight away display the recipe, but using searches will return the index and name of the corresponding recipes, followed by the user inputting the index to view the recipe. I'm not very sure how to implement that well here

Reply:
maybe search can be the main one and then view can be the extension?